packages/fluentui/react-bindings/src/hooks/useIFrameFocusDispatch.ts

```diff
@@ -8,22 +8,30 @@ export const FUI_FRAME_EVENT = 'fuiframefocus';
 export const useIFrameFocusDispatch = (
   enableFrameFocusDispatch: boolean,
   targetDocument: Document,
-  pollDuration: number = 1000,
+  pollDuration: number = 100,
 ) => {
-  const timeoutRef = React.useRef<number>();
-
-  React.useEffect(() => {
-    if (enableFrameFocusDispatch) {
-      timeoutRef.current = targetDocument?.defaultView?.setInterval(() => {
-        const activeElement = targetDocument?.activeElement;
-        if (activeElement?.tagName === 'IFRAME' || activeElement?.tagName === 'WEBVIEW') {
-          const event = new CustomEvent(FUI_FRAME_EVENT, { bubbles: true });
-          activeElement.dispatchEvent(event);
-        }
-      }, pollDuration);
+  React.useEffect((): (() => void) | void => {
+    if (!enableFrameFocusDispatch || !targetDocument) {
+      return;
+    }
+    const { defaultView } = targetDocument;
+    if (!defaultView) {
+      return;
     }
```

Comment:
```suggestion
    const targetWindow = targetDocument.defaultWindow;
    if (!targetWindow) {
      return;
    }
```

Let use meaningful naming